pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/ExpValFunctors.hpp

```diff
@@ -355,4 +265,370 @@ template <class PrecisionT> struct getExpectationValueSparseFunctor {
     }
 };
 
+template <class PrecisionT> struct getExpVal1QubitOpFunctor {
```

Comment:
Here start the code generated with Python, right?

Reply:
It was, but I replace the boilerplate with macro calls. You may think 64 macro calls for the 5Qubit functor is boilerplaty, but it used to be ~1000 lines for that functor alone.

Reply:
Oh my!